code_challenges/code_wars/valid_parentheses.py

```diff
@@ -0,0 +1,22 @@
+def valid_parentheses(characters):
+    open = 0
+    close = 0
+    count = 0
+    for character in characters:
+        if character == "(":
+            open += 1
+            count += 1
+        elif character == ")":
+            close += 1
+            count -= 1
+        if count == -1:
+            return False
+    if open == close:
+        return True
+    else:
+        return False
```

Comment:
If you are returning what the condition evaluates to what is the better return statement here?

Reply:
i did try to use only count before i sent it in but ran into an issue, but i was able to fix it now you requested it  be done.

Reply:
Try "((())"

and "(((())"